cli/src/funTest/kotlin/ExamplesFunTest.kt

```diff
@@ -89,19 +89,19 @@ class ExamplesFunTest : StringSpec() {
             }
         }
 
-        "copyright-garbage.yml can be deserialized" {
+        "The Copyright garbage configuration file can be deserialized" {
```

Comment:
You could scratch "configuration" from several test names, it's not used consistently anyway and it doesn't add value. So just "The copyright garbage file..." is fine IMO.